buf-go.gen.yaml

```diff
@@ -2,5 +2,7 @@ version: v1
 plugins:
   - name: go
```

Comment:
I really like the idea, but we'd need to explicitly import `google.golang.org/grpc/cmd/protoc-gen-go-grpc` somewhere as that's a separate gomod (whose only dependency is `google.golang.org/protobuf`, so it wouldn't be burdensome).

It would be enough to have some file with
```go
//go:build tooling && !tooling

package teleport

import _ "google.golang.org/grpc/cmd/protoc-gen-go-grpc"
```

Should I do it in this PR?

Reply:
Ideally I'd like to try a separate go.mod to hold tools and such, but this doesn't make our go.mod any worse (basically adds a line). So, go for it?

Reply:
To do that change in a meaningful way we'd also need to change the buildbox to remove the protogens, I think, so it should be in a separate PR.

It would be nice to figure out how to not need the buildbox for everything. 😞

Reply:
I'd be OK changing Buf here and the buildbox separately, if you are up for it.

> It would be nice to figure out how to not need the buildbox for everything. 😞

100% yes. Building a million buildboxes every time I bump Buf or some other tool is a huge time sink. That's been on my aim too.